bin/ember

```diff
@@ -27,5 +27,8 @@ resolve('ember-cli', {
     cliArgs: process.argv.slice(2),
     inputStream: process.stdin,
     outputStream: process.stdout
-  }).then(exit);
+  }).then(function(result) {
+    var exitCode = typeof result === 'object' ? result.exitCode : result;
```

Comment:
I'll gladly add a comment explaining what this does and why, with reference to cli.js